morphir/runtime/test/jvm/src/org/finos/morphir/runtime/EvaluatorMDMTests.scala

```diff
@@ -1154,6 +1154,25 @@ object EvaluatorMDMTests extends MorphirBaseSpec {
         testEvalMultiple("diffInDays")("localDateTests", "diffInDaysTest", List(localDate, localDate.plusDays(999)))(
           Data.Int(999)
         ),
+        testEvalMultiple("diffInWeeks")("localDateTests", "diffInWeeksTest", List(localDate, localDate.plusWeeks(2)))(
+          Data.Int(2)
+        ),
+        testEvalMultiple("diffInMonths")(
+          "localDateTests",
+          "diffInMonthsTest",
+          List(localDate, localDate.plusMonths(2))
+        )(
+          Data.Int(2)
+        ),
+        testEvalMultiple("diffInYears")("localDateTests", "diffInYearsTest", List(localDate, localDate.plusYears(2)))(
```

Comment:
I think the tests should specify some edge cases, like the difference between Dec 31 and Jan 1 (honestly not sure if that should be a diff of 0 or of 1)

Reply:
I'll have another PR later for extending LocalTime functions. I'll add a couple of unit tests for these edge cases